packages/react-charting/docs/TestPlans/LineChart/ComponentTests.md

```diff
@@ -0,0 +1,67 @@
+|                      - Mouse over on a line with customized callout                       |                                              Should render customized callout on mouseover                                              |    Enzyme     |
+|                              - Customized callout on a line                               |                                         Should render customized callout correctly on mouseover                                         |    Enzyme     |
+|                - Customized callout on a line from one line to other line                 |                                         Should render customized callout for stack on mouseover                                         |    Enzyme     |
+|               Test 5: Render empty chart aria label div when chart is empty               |                                                                                                                                         |               |
+|                         - Line chart mounted with non-empty data                          |                                                 No empty chart aria label div rendered                                                  |    Enzyme     |
+|                           - Line chart mounted with empty data                            |                                                   Empty chart aria label div rendered                                                   |    Enzyme     |
+|                Test 6: Render empty chart calling with respective to props                |                                                                                                                                         |               |
+| - No prop changes: Mount line chart with non-empty data and then set the same props again |                                              Render function should have been called twice                                              |    Enzyme     |
+|          - prop changes: Mount line chart with empty data and then set the props          |                                             Render function should have been called 3 times                                             |    Enzyme     |
+|                               Test 7: [Sub-Component]: Line                               |                                                                                                                                         |               |
+|                                    - Specify line data                                    |                                                Should render line with the data provided                                                |      RTL      |
+|                            - Hover mouse over the data points                             | Should highlight the data points (No callout is rendered when we hover only on the line. Callout appears on hover over the lines only.) |      RTL      |
```

Comment:
What is this test case? We do show the callout when hovered over the line and not the datapoint.